src/FileLoggerConfiguration.java

```diff
@@ -0,0 +1,54 @@
+public class FileLoggerConfiguration {
+
+    private String filePath;
+    private LoggingLevel loggingLevel;
+    private long maxSizeOfFile;
+    private String format;
+
+    public FileLoggerConfiguration(String filePath, LoggingLevel loggingLevel, long maxSizeOfFile, String format) {
+            this.filePath = filePath;
+            this.loggingLevel = loggingLevel;
+            this.maxSizeOfFile = maxSizeOfFile;
+            this.format = format;
+
+            if (filePath == null || format == null) {
```

Comment:
You've supposed to add validation BEFORE an assignment (`this.filePath = filePath;`), but you did it after for some reason...